webknossos-datastore/app/com/scalableminds/webknossos/datastore/datavault/Encoding.scala

```diff
@@ -1,19 +1,21 @@
 package com.scalableminds.webknossos.datastore.datavault
 
 import com.scalableminds.util.enumeration.ExtendedEnumeration
+import net.liftweb.common.{Box, Failure, Full}
 
+// Lists the encodings supported by VaultPath.readBytes
 object Encoding extends ExtendedEnumeration {
   type Encoding = Value
-  val gzip, brotli, identity, unsupported = Value
+  val gzip, brotli, identity = Value
 
   // Header defined in https://datatracker.ietf.org/doc/html/rfc7231#section-3.1.2.2,
```

Comment:
```suggestion
  // Header defined in https://datatracker.ietf.org/doc/html/rfc7231#section-3.1.2.2
```
just noticed that the link does not work